build-tools/timing/timing.targets

```diff
@@ -63,15 +63,42 @@
     </PropertyGroup>
     <!--Simulate a clean checkout with git commands-->
     <Git
-        Arguments="rm -r --cached %(XACaptureBuildTimingProject.DirectoryToClean)"
+        Condition=" '%(XACaptureBuildTimingProject.Repo)' == '' "
+        Arguments="rm -r --cached %(XACaptureBuildTimingProject.DirectoryFullPath)"
```

Comment:
This should be quoted, in case it contains spaces:

```xml
Arguments="rm -r --cached &quot;%(XACaptureBuildTimingProject.DirectoryFullPath)&quot;"
```